src/components/AssetsTable/index.jsx

```diff
@@ -351,6 +349,22 @@ export class AssetsTable extends React.Component {
         );
         alertType = 'danger';
         break;
+      case assetActions.clear.CLEAR_FILTERS_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.filter.FILTER_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.paginate.PAGE_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.sort.SORT_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
```

Comment:
👍 on this approach to error handling. These failures ought to be infrequent enough; the generic message will be fine